pkg/operator/ceph/object/health.go

```diff
@@ -232,7 +232,7 @@ func (c *bucketChecker) genUserConfig() ObjectUser {
 
 func (c *bucketChecker) testBucketHealth(s3client *S3Agent, bucket string) error {
 	// Purge on exit
-	defer cleanupObjectHealthCheck(s3client, c.objContext.UID)
+	defer cleanupObjectHealthCheck(s3client, c.objContext.UID) //nolint
```

Comment:
Let's change `cleanupObjectHealthCheck()` to log the error, and not return it

Reply:
Done.